### PR TITLE
VACMS-7022 convert legacy theme code to claro theme

### PR DESCRIPTION
**Pull request by tjheffner**
## Description

Relates to #7022 & #6887. Copies existing `vagovadmin` js, scss, and templates to the new `vagovclaro` theme. There are a lot of files in this PR for that reason, but essentially no "new" work. Creating a baseline to refactor from. 

I have commented out many "legacy" styles from being compiled in the new theme, but still included for reference when refactoring. The only template change so far is updated the legacy `fieldset.html.twig` override based on seven to the `fieldset.html.twig` from claro.

## Testing done

- Verified that theme libraries attach to expected pages, helper tooltips & centralized content still behave as expected.
- Verified that the build process correctly transpiles es6 javascript code in the new theme.
- Verified that the gulp process correctly converts sass to css in the new theme.
- Verified that none of the forms look _too wonky_, there are definitely places to clean up. Will be creating tickets today to track these tasks.

## Screenshots
Screen Shot 2021-11-24 at 10 51 07 AM

Screen Shot 2021-11-24 at 11 20 50 AM

## QA steps

As user with admin role,
1. enable vagovclaro as admin theme
2. click around various editor forms and pages, see that no functionality should be changed just different visual styles.

### Definition of Done

- [x] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [x] Automated tests have passed.
- [x] Code Quality Tests have passed.
- [x] Acceptance Criteria in related issue are met.
- [x] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Platform CMS team`
- [x] `Sitewide CMS Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change.
  - [x] Merge & carry on unburdened by announcements

No announcement for Claro work will be needed until we get closer to flipping the switch for all editors/a pilot subset.

### CODE REVIEW

File: docroot/themes/custom/vagovclaro/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# vagovclaro (updated drupal admin theme)
+
+**!! this updated admin theme is currently a work in progress. things subject to change with little notice !!**
+**this is fine because it is not enabled by default, no editors are aware this theme exists. this is a safe space to move fast**
```
Comment (swirtSJW) [inline docroot/themes/custom/vagovclaro/README.md:4]: Love all this documentation.  Thank you.

File: docroot/themes/custom/vagovclaro/gulpfile.es6.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+/**
+ * @file
+ */
+
+/* jshint esversion: 6 */
+
+const gulp = require("gulp");
```
[the remark this answers is not in the document]
Comment (swirtSJW) [reply docroot/themes/custom/vagovclaro/gulpfile.es6.js:7]: I think we'll need to come up with a workaround or these errors will keep resurfacing.  I am hoping @ndouglas might have some ideas.

**Review by swirtSJW — COMMENTED** [review]
All looks good.  Just seeking more info on the ReviewDog notices before merging.

File: docroot/themes/custom/vagovclaro/gulpfile.es6.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (tjheffner) [reply docroot/themes/custom/vagovclaro/gulpfile.es6.js:7]: Do they trigger on current PRs? the gulpfile is the same imports as in vagovadmin.. I'm wondering if once the code is committed will eslint github-actions care in future reviews, if the code isn't changed/surfaced in future PRs?

**Review by swirtSJW — APPROVED** [review]
According to discussion here https://dsva.slack.com/archives/CT4GZBM8F/p1638204228289700 this will not cause future PRs to keep alarming.   So I think we are good to go here.

**Comment by swirtSJW** [comment]
image

**Pull request merged**